api/src/main/java/io/strimzi/api/kafka/model/template/KafkaClusterTemplate.java

```diff
@@ -164,6 +165,16 @@ public void setKafkaContainer(ContainerTemplate kafkaContainer) {
         this.kafkaContainer = kafkaContainer;
     }
 
+    @Description("Template for the Kafka broker TLS sidecar container")
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    public ContainerTemplate getTlsSidecarContainer() {
+        return tlsSidecarContainer;
+    }
+
+    public void setTlsSidecarContainer(ContainerTemplate tlsSidecarContainer) {
```

Comment:
Actually, one more note here ... I think it would be great to be able to configure the env. vars also for the init container. That might be useful to configure some stuff on the Kubernetes client for example.

Reply:
Would that be in the `io.strimzi.kafka.init` package?

Reply:
That is what runs inside the init container. But to add the environment variables would need to be done in `KafkaCluster` in `getInitContainers` I guess.It is basically like the Kafka container or TLS sidecar. Just inserted on other place in the statefulset.

Reply:
Cool, I will add some setting in there.